README.md

```diff
@@ -1,68 +1,69 @@
 
-- Kubernetes version v1.17.0 and above for compatibility. MinIO Operator uses `k8s/client-go` v0.18.0.
-- `kubectl` configured to refer to a Kubernetes cluster.
-- Create the required PVs using [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md).
-- Optional: `kustomize` installed as [explained here](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md#installation).
+To get started, create the MinIO Operator deployment using `kubectl minio` plugin. This is a one time process.
 
-### Create Operator Deployment
+```sh
+kubectl minio operator create
+```
 
-To start MinIO-Operator with default configuration, use the `kubectl apply -k` on this repository.
+Once the MinIO Operator is created, proceed with Tenant creation.
 
-```bash
-kubectl apply -k github.com/minio/operator
-```
+## Tenant Setup
 
-Advanced users can leverage [kustomize](https://github.com/kubernetes-sigs/kustomize) to customize operator configuration via overlays.
+A MinIO Tenant is a MinIO cluster created and managed by the Operator.
 
-```bash
-git clone https://github.com/minio/operator
-kustomize build | kubectl apply -f -
-```
+### Step 1: Create Tenant Namespace
 
-### Create a MinIO Tenant
+Before creating a MinIO Tenant, please create the namespace where this tenant will reside.
 
-Once MinIO-Operator deployment is running, you can create MinIO Tenants using the below command
+For logical isolation, Operator allows a single MinIO Tenant per Kubernetes Namespace.
 
+```sh
+kubectl create ns tenant1-ns
 ```
-kubectl apply -f https://raw.githubusercontent.com/minio/operator/master/examples/tenant.yaml
+
+### Step 2: Create Secret for Tenant Credentials
+
+Next, create the Kubernetes secret that encapsulates root credentials for MinIO Tenant. Please ensure to create secret object with literals `accesskey` and `secretkey`.
+
+Remember to change `YOUR-ACCESS-KEY` and `YOUR-SECRET-KEY` to actual values.
+
+```sh
+kubectl create secret generic tenant1-secret --from-literal=accesskey=YOUR-ACCESS-KEY --from-literal=secretkey=YOUR-SECRET-KEY --namespace tenant1-ns
 ```
 
-### Access Tenant via Service
+Note that the access key and secret key provided here is authorized to perform _all_ operations on the MinIO Tenant.
 
-Add an [external service](https://kubernetes.io/docs/concepts/services-networking/service/) in Tenant definition to enable Service based access to the Tenant pods. Refer [the example here](https://github.com/minio/operator/blob/master/examples/tenant.yaml?raw=true) for details on how to setup service based access for Tenant pods.
+### Step 3: Create MinIO Tenant
 
-### Environment variables
+We can create the tenant now. Before creating the tenant, please ensure you have required nodes and drives in place. Also please ensure that relevant PVs are created. We recommend [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md) to create PVs.
```

Comment:
If we state 'ensure you have the required nodes', it should be clear what is required. Right now based on the command below, what is the ideal k8s infra? I'm guessing 4 nodes and...16 volumes per server? or 16 volumes total. So either 64 or 16 PVCs + the same number of PVs.

Reply:
I added explanation here about the total servers / volumes required @ravindk89